packages/react/etc/react.api.md

```diff
@@ -486,7 +486,7 @@ export class Autofill extends React_2.Component<IAutofillProps, IAutofillState>
     get cursorLocation(): number | null;
     // (undocumented)
     static defaultProps: {
-        enableAutofillOnKeyPress: number[];
+        enableAutofillOnKeyPress: KeyCodes[];
```

Comment:
it seems that this is another breaking change, not sure what `KeyCodes` type is but if it's a union of numbers/strings if user specified a code that is not part of union, they will get type errors

Reply:
it's only breaking for public api md file, d.ts file remains the same, so it was like this before and nothing changes for library users